bigbluebutton-html5/private/locales/pt_BR.json

```diff
@@ -397,7 +397,13 @@
     "app.videoDock.webcamFocusLabel": "Focar",
     "app.videoDock.webcamFocusDesc": "Focar a webcam selecionada",
     "app.videoDock.webcamUnfocusLabel": "Desfocar",
-    "app.videoDock.webcamUnfocusDesc": "Desfocar a webcam selecionada"
-
+    "app.videoDock.webcamUnfocusDesc": "Desfocar a webcam selecionada",
```

Comment:
These translations will have to go through Transifex or will make our life difficult...